<!DOCTYPE html>
<html lang="en">

  <head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.datatables.net/1.10.13/css/jquery.dataTables.min.css" crossorigin="anonymous">
    <!-- <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.1.0/vue.js"></script> -->
    <!-- <script type="text/javascript" src="TBNeoBundle.js"></script> -->
  </head>

  <body>

    <nav class="navbar navbar-toggleable-md navbar-light bg-faded fixed-top">
      <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
	<span class="navbar-toggler-icon"></span>
      </button>

      <a class="navbar-brand" href="#">
	<img src="logo.png" height="30" class="d-inline-block align-top" alt="">
	Reusabledata.org
      </a>
      
      <div class="collapse navbar-collapse" id="navbarsExampleDefault">
        <ul class="navbar-nav mr-auto">
          <li class="nav-item dropdown">
            <a class="nav-link dropdown-toggle" href="http://example.com" id="dropdown01" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Learn more</a>
            <div class="dropdown-menu" aria-labelledby="dropdown01">
              <a class="dropdown-item" href="#home">Home</a>
              <a class="dropdown-item" href="#what-this-is">What this is</a>
              <a class="dropdown-item" href="#who-we-are">Who we are</a>
              <a class="dropdown-item" href="#our-criteria">Our criteria</a>
              <a class="dropdown-item" href="#our-sources-data">Our sources data</a>
              <a class="dropdown-item" href="https://github.com/reusabledata/reusabledata">Contact</a>
            </div>
          </li>
        </ul>
        <form class="form-inline my-2 my-lg-0">
          <input class="form-control mr-sm-2" placeholder="Under construction... :(" type="text">
          <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
        </form>
      </div>
    </nav>
    
    <!-- start jumbotron -->
    <div id="home" class="jumbotron">
      <div class="container">
	<h1 class="display-3">The Reusable Data Project</h1>
	<div class="row">
	  <div class="col-xl-9">
	    <p>
	      Inspired by the efforts of scientists around the world
	      and the game-changing efforts of projects like the
	      Creative Commons, Wikimedia Foundation, and the Free
	      Software movement<!-- (e.g. Debian and the FSF) -->, we
	      hope to engage the larger community in an open and
	      fruitful discussion on issues concerning the use and
	      reuse of scientific data, including the balance of
	      openness and how to make ends meet in an increasingly
	      competitive environment.
	    </p>
	    <p>
	      If you would like to join our efforts to highlight the
	      use and reuse of data in the sciences, please feel free
	      to contact us on
	      our <a href="https://github.com/reusabledata/reusabledata">tracker<a/>
	      or create a pull request against
	      our <a href="https://github.com/reusabledata/reusabledata">repository</a>.
	    </p>
	  </div>
	  <div class="col-xl-3">
	    <img class="mx-auto d-block" style="height:200px;" src="logo.png" />
	  </div>
	</div>
	
	<p>
	  <a class="btn btn-primary btn-lg"
	     href="#what-this-is" role="button">Learn more »</a>
	  &nbsp;&nbsp;&nbsp;
	  <a class="btn btn-success btn-lg"
	     href="#our-sources-data" role="button">See our data »</a>
	</p>	    
      </div>
    </div>
    <!-- end jumbotron -->
    
    <div id="what-this-is" style="height:32px;"></div>
    
    <div class="container">

      <!-- start what-this-is row -->
      <div class="row">

	<div class="col-xl-6">
	  <div class="card">
	    <div class="card-block">
	      <h4 class="card-title">What this is</h4>
	      <p>
		The initial driving concern of this project is the use and
		reuse of biological and biomedical data. However, this is a
		general problem in the scientific community and needs to
		be addressed directly.
		<br />
		For each resource, we attempt to assign one of
		these values objectively by meeting our criteria for
		reusable data:
	      </p>
	      <p>
		<ul>
		  <li>
		    <strong>gold
		    star</strong>&nbsp;<span style="color:#ffd700;font-size:1.5em">&#x2605&nbsp;&#x2605&nbsp;&#x2605&nbsp;&#x2605&nbsp;</span><br />
		    The license unambiguously allows the /free/ reuse
		    and redistribution of the data.
		  </li>
		  <li>
		    <strong>green</strong>&nbsp;<span style="color:green;">&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;</span><br /> The license
		    unambiguously allows reuse and redistribution of the
		    data under some terms.
		  </li>
		  <li>
		    <strong>yellow</strong>&nbsp;<span style="color:orange;">&#x2605;&nbsp;&#x2605;</span><br /> The license is
		    ambiguous or has terms that prevents the reuse and
		    redistribution of in some cases.
		  </li>
		  <li>
		    <strong>red</strong>&nbsp;<span style="color:red;">&#x2605;</span><br /> The license could not be
		    found (so assumed to be under standard copyright) or
		    the license unambiguously makes clear that reuse and
		    redistribution is not welcome for the general
		    community.
		  </li>
		</ul>
	      </p>	      
	      <p>
		If you see any problems with our determinations or would
		like to make corrections or clarifications, please open
		a ticket for us on
		our <a href="https://github.com/reusabledata/reusabledata">issue
		  tracker</a>.
	      </p>
	      <p>
		<a class="btn btn-secondary"
		   href="#who-we-are"
		   role="button">Who we are »</a>
	      </p>
	    </div>
	  </div>
	</div>
        
	<div class="col-xl-6" style="background-color:lightsteelblue;">
	  <p>
	    Inspirational "What this is"-related imagery.
	  </p>
	</div>
	
      </div>
      <!-- end what-this-is row -->

      <!-- spacer -->
      <div id="who-we-are" style="height:32px;"></div>

      <!-- start who we are row -->
      <div class="row">
	
	<div class="col-xl-6" style="background-color:lightcoral;">
	  <p>
	    Inspirational "Who we are"-related imagery.
	  </p>
	</div>
	
	<div class="col-xl-6">
	  <div class="card">
	    <div class="card-block">
	      <h4 class="card-title">Who we are</h4>
	      <p>
		We are not lawyers, this is not legal advice: all
		institutions and groups have their own perspectives
		and counsel. We are a group of scientists,
		engineers, and specialists that are concerned about
		the use and reuse of increasingly interconnected,
		derived, and reprocessed data.
	      </p>
	      <p>
		What we want is to provide a resource looking at some
		of the possible issues with the reuse of scientific
		data.
	      </p>
	      <p>
		We want to actively engage the community in
		conversation and make sure that our information about
		scientific data is up-to-date and correct. If you have
		any questions or see any problems, please open a
		ticket on our tracker.  <br /> If you see an issue,
		please open a ticket on
		our <a href="https://github.com/reusabledata/reusabledata">GitHub
		tracker</a>.
	      </p>
	      <p>
		<a class="btn btn-secondary"
		   href="#our-criteria"
		   role="button">Our criteria »</a>
	      </p>
  	    </div>
	  </div>
	</div>
	
      </div>
      <!-- end who we are row -->
      
      <!-- spacer -->
      <div id="our-criteria" style="height:32px;"></div>

      <div class="row">
	
	<div class="col-xl-8">
	  <div class="card">
	    <div class="card-block">
	      <h4 class="card-title">Our criteria</h4>
	      <p>
		<ol>
		  <li id="criteria-1">
		    <strong>Clearly stated</strong><br /> Have a
		    clearly stated license for data use. In a lot of
		    ways, this is the most important and time-saving
		    point. If there is no license, then rights are
		    unclear and one needs to assume the
		    default:
		    <a href="https://www.copyright.gov/help/faq/faq-general.html">all rights reserved</a>.
		  </li>
		  <li id="criteria-2">
		    <strong>Allow use and reuse</strong><br /> Data
		    that cannot be used is not data at all. In modern
		    data-driven science, the ability to synthesize
		    data and make the results available for others to
		    do the same is paramount.
		  </li>
		  <li id="criteria-3">
		    <strong>Non-discriminatory</strong><br /> Ideally,
		    data should be able to be used and reused by
		    anybody for any purpose, academic, commercial, and
		    anything else.
		  </li>
		  <li id="criteria-4">
		    <strong>Non-revocable</strong><br /> Of course
		    somebody might choose to close there data off in
		    the future, but there should be no language that
		    revokes usage rights retroactively for the data
		    before that horizon.
		  </li>
		  <li id="criteria-5">
		    <strong>Freely and openly available</strong><br />
		    (Gold stars) The data should should be available to all people for
		    any purpose. An example of this would be data
		    explicitly released under the <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons 4.0
		      Attribution 4.0 International license</a>.
		  </li>
		</ol>
	      </p>
	      <p>
		<a class="btn btn-secondary"
		   href="#our-sources-data"
		   role="button">Our sources data »</a>
	      </p>
	    </div>
	  </div>
	</div>
	
	<div class="col-xl-4" style="background-color:mediumorchid;">
	  <p>
	    Inspirational "Our criteria"-related imagery.
	  </p>
	</div>
	
      </div> <!-- end row -->
      
      <!-- spacer -->
      <div id="our-sources-data" style="height:32px;"></div>

      <div class="row">

	<div class="col-xl-12">
	  <div class="card">
	    <div class="card-block">
	      <h4 class="card-title">Our sources data</h4>
	      <p class="card-text">
		
		<table class="table table-striped" id="sourcestable"><thead><tr><th>Name</th><th>Categories</th><th>Grade</th><th>Description</th><th>License Info</th><th>License Issues</th></tr></thead><tfoot><tr><th>Name</th><th>Categories</th><th>Grade</th><th>Description</th><th>License Info</th><th>License Issues</th></tr></tfoot><tbody><tr><td> <a href="http://bgee.org">BGee</a></td><td>x-species mouse expression data</td><!--td(class='bg-danger text-white') &#x2605;--><td><span style="color:red;">&#x2605;</span></td><td>Bgee is a database to retrieve and compare gene expression patterns in multiple animal species, produced from multiple data types (RNA-Seq, Affymetrix, in situ hybridization, and EST data).</td><td><a href="https://github.com/BgeeDB/bgee/blob/master/LICENSE">copyleft</a></td><td><dl><dt><!-- a(href='#criteria-' + issue.criteria) Criteria&nbsp;#{issue.criteria}--><a href="#our-criteria">Criteria&nbsp;1</a></dt><dd><small class="text-muted">We are unsure whether the data is covered under the GPLv3 or has all rights reserved.</small></dd></dl><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="It is unclear whether the data or the software is the listed GNU GPL v.3.0; if the former, it may be problematic, if the latter, all rights may be reserved. They do not seem to me looking at the licenses of their own data (http://bgee.org/?page=source).">Commentary</button></td></tr><tr><td> <a>BioGRID</a></td><td>biology x-species protein-protein interaction</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>interaction</td><td><a href="http://wiki.thebiogrid.org/doku.php/terms_and_conditions">source specific</a></td><td><span></span></td></tr><tr><td> <a>ClinVar</a></td><td>biology human disease-gene association,variant-disease association,variant definitions</td><td style="color:#ffd700;font-size:1.5em;">&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;</td><td>ClinVar archives and aggregates information about relationships among variation and human health. ClinVar collects reports of variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data.</td><td><a href="http://www.ncbi.nlm.nih.gov/clinvar/docs/maintenance_use/">public domain</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="TODO: Need to square the reuse on non-PD information.">Commentary</button></td></tr><tr><td> <a>Coriell Institute for Medical Research</a></td><td>biology human disease-model association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>The Coriell Cell Repositories provide essential research reagents to the scientific community by establishing, verifying, maintaining, and distributing cell cultures and DNA derived from cell cultures. These collections, supported by funds from the National Institutes of Health (NIH) and several foundations, are extensively utilized by research scientists around the world. NINDS and NIGMS cell line catalog. NIGMS samples represent a variety of disease states, chromosomal abnormalities, apparently healthy individuals and many distinct human populations. NINDS samples are drawn from subjects with cerebrovascular disease, epilepsy, motor neuron disease, Parkinsonism and Tourette Syndrome, as well as controls.</td><td><a href="https://www.coriell.org/legal-notice">undetermined</a></td><td><span></span></td></tr><tr><td> <a>Comparative Toxicogenomics Database (CTD)</a></td><td>biology x-species disease-gene association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>CTD promotes understanding about the effects of environmental chemicals on human health by integrating data from curated scientific literature to describe chemical interactions with genes and proteins, and associations between diseases and chemicals, and diseases and genes/proteins.</td><td><a href="http://ctdbase.org/about/legal.jsp">undetermined</a></td><td><span></span></td></tr><tr><td> <a>ENCODE</a></td><td>biology genomic resource genomic elements</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels, and regulatory elements that control cells and circumstances in which a gene is active.</td><td><a href="https://www.genome.gov/10005107/#al-5">undetermined</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="After 2014 policy: OK.&lt;br /&gt;2008-2014 policy: &quot;To facilitate this compromise between unrestricted use of the data and unavailability of the data until publication, NHGRI will promote observation of a 9-month period during which resource users may freely use the ENCODE/modENCODE data to design and carry out their own research programs, but not to submit publications or make public presentations that use unpublished ENCODE/modENCODE data without prior consent. The publication and presentation moratorium is expected to extend to all forms of public disclosure, including meeting abstracts, oral presentations, and formal electronic submissions to publicly accessible sites (e.g., public websites, web blogs). (NHGRI recognizes that there may be some exceptions to this blanket request; examples are discussed more fully below.)&quot;">Commentary</button></td></tr><tr><td> <a>Fantom5</a></td><td>biology human gene expression</td><td style="color:#ffd700;font-size:1.5em;">&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;</td><td>We are complex multicellular organisms composed of ~400 distinct cell types. This diversity of cell types allow us to see, think, hear, fight infections etc. yet all of this is encoded in the same genome. The difference between all these cells is what parts of the genome they use – for instance, neurons use different genes than muscle cells, and therefore they work very differently. In FANTOM5, we have systematically investigated exactly what are the sets of genes used in virtually all cell types across the human body, and the genomic regions which determine where the genes are read from. We aim to use this information to build transcriptional regulatory models for every primary cell type that makes up a human.</td><td><a href="http://fantom.gsc.riken.jp/5/">CC4</a></td><td><span></span></td></tr><tr><td> <a>FlyBase</a></td><td>biology MOD genotype-phenotype association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>FlyBase is the model organism database providing integrated genetic, genomic, phenomic, and biological data for Drosophila melanogaster.</td><td><a href="http://flybase.org/wiki/FlyBase:About#FlyBase_Copyright">undetermined</a></td><td><span></span></td></tr><tr><td> <a>GTEx</a></td><td>biology human gene expression</td><!--td(class='bg-danger text-white') &#x2605;--><td><span style="color:red;">&#x2605;</span></td><td>The Genotype-Tissue Expression (GTEx) project aims to provide to the scientific community a resource with which to study human gene expression and regulation and its relationship to genetic variation. This project will collect and analyze multiple human tissues from donors who are also densely genotyped, to assess genetic variation within their genomes. By analyzing global RNA expression within individual tissues and treating the expression levels of genes as quantitative traits, variations in gene expression that are highly correlated with genetic variation can be identified as expression quantitative trait loci, or eQTLs.</td><td><a href="http://www.gtexportal.org/home/documentationPage#staticTextPublicationPolicy">undetermined</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="Apparently, the data not under the exceptions listed are under the NIH GDS (https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html), which appears to be a well-thought out, but restrictive regime. The NIH GDS section V part B terms are quite restrictive.">Commentary</button></td></tr><tr><td> <a>Human Phenotype Ontology (HPO)</a></td><td>biology human disease-phenotype association</td><!--td(class='bg-warning text-white') &#x2605;&nbsp;&#x2605;--><td><span style="color:orange;">&#x2605;&nbsp;&#x2605;</span></td><td>A curated database of human hereditary syndromes from OMIM, Orphanet, and DECIPHER mapped to classes of the human phenotype ontology. Various meta-attributes such as frequency, references and negations are associated with each annotation. These are presently limited to rare mendelian diseases.</td><td><a href="http://human-phenotype-ontology.github.io/license.html">undetermined</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="immutability&lt;br /&gt;unclear derivatives">Commentary</button></td></tr><tr><td> <a>International Mouse Phenotyping Consortium (IMPC)</a></td><td>biology mouse genotype-phenotype association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>The International Mouse Phenotyping Consortium (IMPC) is generating a knockout mouse strain for every protein coding gene by using the embryonic stem cell resource generated by the International Knockout Mouse Consortium (IKMC). Systematic broad-based phenotyping is performed by each IMPC center using standardized procedures found within the International Mouse Phenotyping Resource of Standardised Screens (IMPReSS) resource. Gene-to-phenotype associations are made by a versioned statistical analysis.</td><td><a href="http://www.mousephenotype.org/about-impc/additional-information/data-use-agreement">undetermined</a></td><td><span></span></td></tr><tr><td> <a>Kyoto Encyclopedia of Genes and Genomes (KEGG)</a></td><td>biology genomic resource gene-pathway association,disease-gene association,orthology</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>KEGG is an integrated database resource consisting of the seventeen main databases including systems, genomic, chemical, and health information.</td><td><a href="http://www.kegg.jp/kegg/legal.html">undetermined</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="Pathway Solutions here: http://www.pathway.jp/">Commentary</button></td></tr><tr><td> <a>Mouse Genome Informatics (MGI)</a></td><td>biology MOD genotype-phenotype association,disease-model association,gene expression</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>MGI is the international database resource for the laboratory mouse, providing integrated genetic, genomic, and biological data to facilitate the study of human health and disease.</td><td><a href="http://www.informatics.jax.org/mgihome/other/copyright.shtml">undetermined</a></td><td><span></span></td></tr><tr><td> <a>Mouse Phenome Database (MPD)</a></td><td>biology MOD genotype (strain)-phenotype association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>The Mouse Phenome Database is a collaborative standardized collection of measured data on laboratory mouse strains, and includes: baseline phenotype data sets; studies of drug, diet, disease and aging effect; protocols, projects, and publications; and SNP, variation and gene expression studies. MPD collects data for classical inbred strains, other fixed-genotype strains, derived lines and populations that are openly acquirable (strain panel examples). Strains can be from JAX-Mice or from any other vendor that's a recognized breeding source.</td><td><a href="http://phenome.jax.org/db/q?rtn=docs/termsofuse">undetermined</a></td><td><span></span></td></tr><tr><td> <a>MyGene.info</a></td><td>biology genomic resource gene definition</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>MyGene.info provides a simple-to-use REST web services to query/retrieve gene annotation data.</td><td><a href="TODO: http://docs.mygene.info/en/v3/terms.html vs. http://mygene.info/terms/">undetermined</a></td><td><span></span></td></tr><tr><td> <a>National Center for Biotechnology Information Gene (NCBI-Gene)</a></td><td>biology genomic resource gene definition,taxon definition,gene-publication association</td><td style="color:#ffd700;font-size:1.5em;">&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;</td><td>NCBI Gene integrates information from a wide range of species, and includes nomenclature, Reference Sequences (RefSeqs), maps, pathways, variations, phenotypes, and links to genome-, phenotype-, and locus-specific resources worldwide. Taxon lists the taxinomic organization of organisms. Pub2Gene serves links between genes and pubmed identifiers where they are mentioned.</td><td><a href="http://www.ncbi.nlm.nih.gov/home/about/policies.shtml">undetermined</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="TODO: Need to square the reuse on non-PD information.">Commentary</button></td></tr><tr><td> <a>Online Mendelian Inheritance in Animals (OMIA)</a></td><td>biology veterinary x-species gene-disease association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>Online Mendelian Inheritance in Animals (OMIA) is a catalogue/compendium of inherited disorders, other (single-locus) traits, and genes in 215 (non-model) animal species.</td><td><a href="http://sydney.edu.au/disclaimer.html">undetermined</a></td><td><span></span></td></tr><tr><td> <a>Mendelian Inheritance in Man relationship to HPO construction and maintenance (OMIM)</a></td><td>biology human disease-phenotype association,gene-disease association,variant-disease association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes with full-text, referenced overviews that contains information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.</td><td><a href="http://www.omim.org/help/agreement">undetermined</a></td><td><span></span></td></tr><tr><td> <a>ORPHANET</a></td><td>biology human disease-gene association,disease-phenotype association,ontology</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>Orphanet provides reference information on rare diseases and orphan drugs to help improve the diagnosis, care and treatment of patients with rare diseases.</td><td><a href="http://www.orpha.net/consor/cgi-bin/Education_AboutOrphanet.php?lng=EN&amp;stapage=CGU">undetermined</a></td><td><span></span></td></tr><tr><td> <a>Protein ANalysis THrough Evolutionary Relationships Classification System (PANTHER)</a></td><td>biology genomic resource orthology</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) according to evolutionary family/subfamily, molecular function, biological process, and pathway. The PANTHER Classifications are the result of human curation as well as sophisticated bioinformatics algorithms.</td><td><a href="http://pantherdb.org/terms/disclaimer.jsp ?">ARR</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="TODO: Reach out to Paul--according to the site he is the sole copyright holder. Considering the disclaimer, I believe the intention is something BSD-ish, but the literal read is all right reserved.">Commentary</button></td></tr><tr><td> <a>Reactome</a></td><td>biology pathway x-species pathway data</td><td style="color:#ffd700;font-size:1.5em;">&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;&nbsp;&#x2605;</td><td>Reactome is a free, open-source, curated and peer reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education.</td><td><a href="http://www.reactome.org/pages/about/license-agreement/">CC4</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="There is a comment in there about KEGG data in Reactome. Thus Reactome FI Network data is not available.">Commentary</button></td></tr><tr><td> <a>WormBase database of nematode biology (WormBase)</a></td><td>biology MOD allele-phenotype association</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>WormBase is an international consortium dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of C. elegans and related nematodes.</td><td><a href="http://www.wormbase.org/about/policies#0--10">undetermined</a></td><td><button class="btn btn-secondary btn-sm" role="button" data-toggle="popover" title="License commentary" data-placement="left" data-content="This has to do with hitting their servers to much and being too big a load; this may not apply.">Commentary</button></td></tr><tr><td> <a>Zebrafish Information Network (ZFIN)</a></td><td>biology MOD genotype-phenotype association,experimental reagents (morpholinos)</td><!--td(class='bg-inverse text-white') ?--><td><strong>?</strong></td><td>The Zebrafish Information Resource is the community database resource for the laboratory use of zebrafish which develops and supports integrated zebrafish genetic, genomic and developmental information, maintains the definitive reference data sets of zebrafish research information toward facilitation of the use of zebrafish as a model for human biology.</td><td><a href="http://zfin.org/warranty.html">undetermined</a></td><td><span></span></td></tr></tbody></table>
	
		<!-- <table id="sourcestable" class="table table-striped"> -->
		<!--   <thead> -->
		<!--     <tr> -->
		<!--       <th>Name</th> -->
		<!--       <th>Categories</th> -->
		<!--       <th>Grade</th> -->
		<!--       <th>Description</th> -->
		<!--       <th>License Info</th> -->
		<!--       <th>License Issues</th> -->
		<!--     </tr> -->
		<!--   </thead> -->
		<!--   <tfoot> -->
		<!--     <tr> -->
		<!--       <th>Name</th> -->
		<!--       <th>Categories</th> -->
		<!--       <th>Grade</th> -->
		<!--       <th>Description</th> -->
		<!--       <th>License Info</th> -->
		<!--       <th>License Issues</th> -->
		<!--     </tr> -->
		<!--   </tfoot> -->
		<!--   <tbody> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>BGee</td> -->
		<!--       <td> -->
		<!-- 	x-species -->
		<!-- 	<br /> -->
		<!-- 	mouse -->
		<!-- 	<br /> -->
		<!-- 	expression data -->
		<!--       </td> -->
		<!--       <td>red</td> -->
		<!--       <td>Bgee is a database to retrieve and compare gene expression patterns in multiple animal species, produced from multiple data types (RNA-Seq, Affymetrix, in situ hybridization, and EST data).</td> -->
		<!--       <td> -->
		<!-- 	<a href="https:&#x2F;&#x2F;github.com&#x2F;BgeeDB&#x2F;bgee&#x2F;blob&#x2F;master&#x2F;LICENSE">copyleft</a> -->
		<!--       </td> -->
		<!--       <td></td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>BioGRID</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	x-species -->
		<!-- 	<br /> -->
		<!-- 	protein-protein interaction -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>interaction</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;wiki.thebiogrid.org&#x2F;doku.php&#x2F;terms_and_conditions">source specific</a> -->
		<!--       </td> -->
		<!--       <td>All information, data, text, software, music, sound, photographs, graphics, video, messages, or any other materials whatsoever (collectively, “Content”), whether publicly posted or privately transmitted, is the sole responsibility of the person from whom such Content originated. This means that the user, and not BioGRID, is entirely responsible for all Content that he or she uploads, posts, emails or otherwise transmits via the BioGRID Sites.,Use, reproduction, modification, and other intellectual property rights to data stored on the BioGRID Sites will be subject to licensing arrangements that may be approved by BioGRID as applicable to such Content.,You agree not to sell, resell, or offer for any commercial purposes, any portion of the BioGRID Sites, use of the BioGRID Sites, or access to the BioGRID Sites.,Our designated agent to receive notification of claimed infringement is:  Mr. Terry Donaghue Director Technology Transfer and Industrial Liason 600 University Avenue Toronto, ON, Canada donaghue@mshri.on.ca TEL: 416-586-8225 FAX: 416-586-3110</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>ClinVar</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	human -->
		<!-- 	<br /> -->
		<!-- 	disease-gene association,variant-disease association,variant definitions -->
		<!--       </td> -->
		<!--       <td>gold</td> -->
		<!--       <td>ClinVar archives and aggregates information about relationships among variation and human health. ClinVar collects reports of variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;clinvar&#x2F;docs&#x2F;maintenance_use&#x2F;">public domain</a> -->
		<!--       </td> -->
		<!--       <td>NOTE: This site contains resources which incorporate material contributed or licensed by individuals, companies, or organizations that may be protected by U.S. and foreign copyright laws. These include, but are not limited to PubMed Health (see PubMed Health Copyright and Restrictions Notice), PubMed Central (PMC) (see PMC Copyright Notice), Bookshelf (see Bookshelf Copyright Notice), OMIM (see OMIM Copyright Status), and PubChem. All persons reproducing, redistributing, or making commercial use of this information are expected to adhere to the terms and conditions asserted by the copyright holder. Transmission or reproduction of protected items beyond that allowed by fair use (PDF) as defined in the copyright laws requires the written permission of the copyright owners.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Coriell Institute for Medical Research</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	human -->
		<!-- 	<br /> -->
		<!-- 	disease-model association -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>The Coriell Cell Repositories provide essential research reagents to the scientific community by establishing, verifying, maintaining, and distributing cell cultures and DNA derived from cell cultures. These collections, supported by funds from the National Institutes of Health (NIH) and several foundations, are extensively utilized by research scientists around the world. NINDS and NIGMS cell line catalog. NIGMS samples represent a variety of disease states, chromosomal abnormalities, apparently healthy individuals and many distinct human populations. NINDS samples are drawn from subjects with cerebrovascular disease, epilepsy, motor neuron disease, Parkinsonism and Tourette Syndrome, as well as controls.</td> -->
		<!--       <td> -->
		<!-- 	<a href="https:&#x2F;&#x2F;www.coriell.org&#x2F;legal-notice">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>Material on the Coriell Institute website is protected by copyright law. Unless otherwise stated, users may print or download information from this site for personal, non-commercial use only, provided they identify the source of the material (Coriell Institute for Medical Research) and include a statement that the materials are protected by copyright law. Permission to reprint or otherwise reproduce any document in whole or in part is prohibited, unless prior written consent is obtained from the copyright owner (email request to: communications(at)coriell.org).</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Comparative Toxicogenomics Database (CTD)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	x-species -->
		<!-- 	<br /> -->
		<!-- 	disease-gene association -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>CTD promotes understanding about the effects of environmental chemicals on human health by integrating data from curated scientific literature to describe chemical interactions with genes and proteins, and associations between diseases and chemicals, and diseases and genes&#x2F;proteins.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;ctdbase.org&#x2F;about&#x2F;legal.jsp">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>Any reproduction or use for commercial purpose is prohibited without the prior express written permission of the MDI Biological Laboratory and NC State University.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>ENCODE</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	genomic resource -->
		<!-- 	<br /> -->
		<!-- 	genomic elements -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels, and regulatory elements that control cells and circumstances in which a gene is active.</td> -->
		<!--       <td> -->
		<!-- 	<a href="https:&#x2F;&#x2F;www.genome.gov&#x2F;10005107&#x2F;#al-5">undetermined</a> -->
		<!--       </td> -->
		<!--       <td></td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Fantom5</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	human -->
		<!-- 	<br /> -->
		<!-- 	gene expression -->
		<!--       </td> -->
		<!--       <td>gold</td> -->
		<!--       <td>We are complex multicellular organisms composed of ~400 distinct cell types. This diversity of cell types allow us to see, think, hear, fight infections etc. yet all of this is encoded in the same genome. The difference between all these cells is what parts of the genome they use – for instance, neurons use different genes than muscle cells, and therefore they work very differently. In FANTOM5, we have systematically investigated exactly what are the sets of genes used in virtually all cell types across the human body, and the genomic regions which determine where the genes are read from. We aim to use this information to build transcriptional regulatory models for every primary cell type that makes up a human.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;fantom.gsc.riken.jp&#x2F;5&#x2F;">CC4</a> -->
		<!--       </td> -->
		<!--       <td></td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>FlyBase</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	MOD -->
		<!-- 	<br /> -->
		<!-- 	genotype-phenotype association -->
		<!--       </td> -->
		<!--       <td>ugly</td> -->
		<!--       <td>FlyBase is the model organism database providing integrated genetic, genomic, phenomic, and biological data for Drosophila melanogaster.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;flybase.org&#x2F;wiki&#x2F;FlyBase:About#FlyBase_Copyright">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>This publication may be copied for non-commercial, scientific uses by individuals or organizations (including for-profit organizations). FlyBase is freely distributed to the scientific community on the understanding that it will not be used for commercial gain by any organization. Any commercial use of this publication, or any parts thereof, is expressly prohibited without permission in writing from the FlyBase consortium.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>GTEx</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	human -->
		<!-- 	<br /> -->
		<!-- 	gene expression -->
		<!--       </td> -->
		<!--       <td>red</td> -->
		<!--       <td>The Genotype-Tissue Expression (GTEx) project aims to provide to the scientific community a resource with which to study human gene expression and regulation and its relationship to genetic variation. This project will collect and analyze multiple human tissues from donors who are also densely genotyped, to assess genetic variation within their genomes. By analyzing global RNA expression within individual tissues and treating the expression levels of genes as quantitative traits, variations in gene expression that are highly correlated with genetic variation can be identified as expression quantitative trait loci, or eQTLs.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.gtexportal.org&#x2F;home&#x2F;documentationPage#staticTextPublicationPolicy">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>All datasets from phs00424.v5.p1 forward will follow the NIH GDS policy. This means that once released through dbGaP, there are no restrictions on use or publication. This document and an accompanying table of dataset releases can be found at http:&#x2F;&#x2F;www.gtexportal.org&#x2F;home&#x2F;documentationPage.,https:&#x2F;&#x2F;grants.nih.gov&#x2F;grants&#x2F;guide&#x2F;notice-files&#x2F;NOT-OD-14-124.html</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Human Phenotype Ontology (HPO)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	human -->
		<!-- 	<br /> -->
		<!-- 	disease-phenotype association -->
		<!--       </td> -->
		<!--       <td>yellow</td> -->
		<!--       <td>A curated database of human hereditary syndromes from OMIM, Orphanet, and DECIPHER mapped to classes of the human phenotype ontology. Various meta-attributes such as frequency, references and negations are associated with each annotation. These are presently limited to rare mendelian diseases.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;human-phenotype-ontology.github.io&#x2F;license.html">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>That the Human Phenotype Ontology Consortium is acknowledged and cited properly.,That neither the content of the HPO file(s) nor the logical relationships embedded within the HPO file(s) be altered in any way.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>International Mouse Phenotyping Consortium (IMPC)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	mouse -->
		<!-- 	<br /> -->
		<!-- 	genotype-phenotype association -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>The International Mouse Phenotyping Consortium (IMPC) is generating a knockout mouse strain for every protein coding gene by using the embryonic stem cell resource generated by the International Knockout Mouse Consortium (IKMC). Systematic broad-based phenotyping is performed by each IMPC center using standardized procedures found within the International Mouse Phenotyping Resource of Standardised Screens (IMPReSS) resource. Gene-to-phenotype associations are made by a versioned statistical analysis.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.mousephenotype.org&#x2F;about-impc&#x2F;additional-information&#x2F;data-use-agreement">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>When the first major Project paper on these analyses is published, then researchers inside and outside the Project are free to present and publish using the Project data for these and other analyses.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Kyoto Encyclopedia of Genes and Genomes (KEGG)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	genomic resource -->
		<!-- 	<br /> -->
		<!-- 	gene-pathway association,disease-gene association,orthology -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>KEGG is an integrated database resource consisting of the seventeen main databases including systems, genomic, chemical, and health information.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.kegg.jp&#x2F;kegg&#x2F;legal.html">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>Non-academic users and Academic users intending to use KEGG for commercial purposes are requested to obtain a license agreement through KEGG&#39;s exclusive licensing agent, Pathway Solutions, for installation of KEGG at their sites, for distribution or reselling of KEGG data, for software development or any other commercial activities that make use of KEGG, or as end users of any third-party application that requires downloading of KEGG data or access to KEGG data via the KEGG API.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Mouse Genome Informatics (MGI)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	MOD -->
		<!-- 	<br /> -->
		<!-- 	genotype-phenotype association,disease-model association,gene expression -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>MGI is the international database resource for the laboratory mouse, providing integrated genetic, genomic, and biological data to facilitate the study of human health and disease.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.informatics.jax.org&#x2F;mgihome&#x2F;other&#x2F;copyright.shtml">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>This software and data are provided to enhance knowledge and encourage progress in the scientific community and are to be used only for research and educational purposes.,Any reproduction or use for commercial purpose is prohibited without the prior express written permission of the Jackson Laboratory.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Mouse Phenome Database (MPD)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	MOD -->
		<!-- 	<br /> -->
		<!-- 	genotype (strain)-phenotype association -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>The Mouse Phenome Database is a collaborative standardized collection of measured data on laboratory mouse strains, and includes: baseline phenotype data sets; studies of drug, diet, disease and aging effect; protocols, projects, and publications; and SNP, variation and gene expression studies. MPD collects data for classical inbred strains, other fixed-genotype strains, derived lines and populations that are openly acquirable (strain panel examples). Strains can be from JAX-Mice or from any other vendor that&#39;s a recognized breeding source.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;phenome.jax.org&#x2F;db&#x2F;q?rtn&#x3D;docs&#x2F;termsofuse">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>MPD data contributors waive copyright and related rights, making their data freely available whereby MPD users may freely build upon, enhance and reuse those data for any purpose without restriction. Scholarly citation norms must be followed for content reuse.,The Jackson Laboratory&#39;s web site privacy policy applies to all use of this resource.,The Mouse Phenome Database (MPD) is an NIH-funded data sharing repository and adheres to the NIH Genomic Data Sharing Policy.,Any reproduction of JAX(R) Mice mouse images is prohibited without prior express written permission (see the JAX(R) Mice Terms of Use for details).,Data contributors and data users may not use MPD in any unlawful manner, or in any manner that would impair MPD&#39;s services, security or functionality. Automated usage (webcrawlers and similar) must observe each page&#39;s &quot;meta robots&quot; html tags and space requests by ≥ 2 seconds. We reserve the right to block violators and to take any other action as we deem necessary.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>MyGene.info</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	genomic resource -->
		<!-- 	<br /> -->
		<!-- 	gene definition -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>MyGene.info provides a simple-to-use REST web services to query&#x2F;retrieve gene annotation data.</td> -->
		<!--       <td> -->
		<!-- 	<a href="TODO: http:&#x2F;&#x2F;docs.mygene.info&#x2F;en&#x2F;v3&#x2F;terms.html vs. http:&#x2F;&#x2F;mygene.info&#x2F;terms&#x2F;">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>You also agree that you will ensure that any copies of documents you generate by using MyGene.info shall retain and display all copyright and other proprietary notices contained therein.,Nothing contained herein shall be construed as conferring by implication, estoppel or otherwise any license or right under any patent, trademark or other intellectual property of TSRI or any third party. Except as expressly provided above, nothing contained herein shall be construed as conferring any right or license under any TSRI copyrights.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>National Center for Biotechnology Information Gene (NCBI-Gene)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	genomic resource -->
		<!-- 	<br /> -->
		<!-- 	gene definition,taxon definition,gene-publication association -->
		<!--       </td> -->
		<!--       <td>gold</td> -->
		<!--       <td>NCBI Gene integrates information from a wide range of species, and includes nomenclature, Reference Sequences (RefSeqs), maps, pathways, variations, phenotypes, and links to genome-, phenotype-, and locus-specific resources worldwide. Taxon lists the taxinomic organization of organisms. Pub2Gene serves links between genes and pubmed identifiers where they are mentioned.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.ncbi.nlm.nih.gov&#x2F;home&#x2F;about&#x2F;policies.shtml">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>NOTE: This site contains resources which incorporate material contributed or licensed by individuals, companies, or organizations that may be protected by U.S. and foreign copyright laws. These include, but are not limited to PubMed Health (see PubMed Health Copyright and Restrictions Notice), PubMed Central (PMC) (see PMC Copyright Notice), Bookshelf (see Bookshelf Copyright Notice), OMIM (see OMIM Copyright Status), and PubChem. All persons reproducing, redistributing, or making commercial use of this information are expected to adhere to the terms and conditions asserted by the copyright holder. Transmission or reproduction of protected items beyond that allowed by fair use (PDF) as defined in the copyright laws requires the written permission of the copyright owners.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Online Mendelian Inheritance in Animals (OMIA)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	veterinary x-species -->
		<!-- 	<br /> -->
		<!-- 	gene-disease association -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>Online Mendelian Inheritance in Animals (OMIA) is a catalogue&#x2F;compendium of inherited disorders, other (single-locus) traits, and genes in 215 (non-model) animal species.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;sydney.edu.au&#x2F;disclaimer.html">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>Individuals may copy a reasonable portion of the material on this site under the fair dealing provisions of the Copyright Act 1968 provided it is used for research or study or for criticism or review. Educational institutions may reproduce limited amounts of material from this site under Part VB of the Copyright Act,Apart from these specified uses material from this site may not be reproduced for commercial purposes without prior written permission of the University of Sydney.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Mendelian Inheritance in Man relationship to HPO construction and maintenance (OMIM)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	human -->
		<!-- 	<br /> -->
		<!-- 	disease-phenotype association,gene-disease association,variant-disease association -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes with full-text, referenced overviews that contains information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.omim.org&#x2F;help&#x2F;agreement">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>Use of OMIM.org is provided free of charge to any individual for personal use, for educational or scholarly use, or for research purposes through the front end of the database.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>ORPHANET</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	human -->
		<!-- 	<br /> -->
		<!-- 	disease-gene association,disease-phenotype association,ontology -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>Orphanet provides reference information on rare diseases and orphan drugs to help improve the diagnosis, care and treatment of patients with rare diseases.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.orpha.net&#x2F;consor&#x2F;cgi-bin&#x2F;Education_AboutOrphanet.php?lng&#x3D;EN&amp;stapage&#x3D;CGU">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>No element or part of the site (including, but not limited to, the database and the information contained within) can be copied, translated, reproduced, exploited, published, electronically stocked or redistributed for commercial use without prior written consent.,Non-profit organisations are authorised to copy and distribute information from the site, as long as this information is neither modified nor sold. In addition to this, the terms of the intellectual property rights and the source of the information must be provided.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Protein ANalysis THrough Evolutionary Relationships Classification System (PANTHER)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	genomic resource -->
		<!-- 	<br /> -->
		<!-- 	orthology -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) according to evolutionary family&#x2F;subfamily, molecular function, biological process, and pathway. The PANTHER Classifications are the result of human curation as well as sophisticated bioinformatics algorithms.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;pantherdb.org&#x2F;terms&#x2F;disclaimer.jsp ?">ARR</a> -->
		<!--       </td> -->
		<!--       <td></td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Reactome</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	pathway x-species -->
		<!-- 	<br /> -->
		<!-- 	pathway data -->
		<!--       </td> -->
		<!--       <td>gold</td> -->
		<!--       <td>Reactome is a free, open-source, curated and peer reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.reactome.org&#x2F;pages&#x2F;about&#x2F;license-agreement&#x2F;">CC4</a> -->
		<!--       </td> -->
		<!--       <td></td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>WormBase database of nematode biology (WormBase)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	MOD -->
		<!-- 	<br /> -->
		<!-- 	allele-phenotype association -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>WormBase is an international consortium dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of C. elegans and related nematodes.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;www.wormbase.org&#x2F;about&#x2F;policies#0--10">undetermined</a> -->
		<!--       </td> -->
		<!--       <td></td> -->
		<!--     </tr> -->
		<!--      -->
		<!--     <tr> -->
		<!--       <td>Zebrafish Information Network (ZFIN)</td> -->
		<!--       <td> -->
		<!-- 	biology -->
		<!-- 	<br /> -->
		<!-- 	MOD -->
		<!-- 	<br /> -->
		<!-- 	genotype-phenotype association,experimental reagents (morpholinos) -->
		<!--       </td> -->
		<!--       <td>undetermined</td> -->
		<!--       <td>The Zebrafish Information Resource is the community database resource for the laboratory use of zebrafish which develops and supports integrated zebrafish genetic, genomic and developmental information, maintains the definitive reference data sets of zebrafish research information toward facilitation of the use of zebrafish as a model for human biology.</td> -->
		<!--       <td> -->
		<!-- 	<a href="http:&#x2F;&#x2F;zfin.org&#x2F;warranty.html">undetermined</a> -->
		<!--       </td> -->
		<!--       <td>ZFIN Software and Data are provided to enhance knowledge and encourage progress in the scientific community, and are to be used only for research and educational purposes. ZFIN reserves all rights not expressly granted. Further, ZFIN retains all rights, title, and interest in and to the Software and Data. Any reproduction or use for a commercial purpose is prohibited without the prior express written permission of ZFIN. Distribution of the Software or Data to any third party, without a separate written agreement with ZFIN, is prohibited.</td> -->
		<!--     </tr> -->
		<!--      -->
		<!--   </tbody> -->
		<!-- </table>	 -->
		
	      </p>
	      <p>
		<a class="btn btn-secondary"
		   href="https://github.com/reusabledata/reusabledata"
		   role="button">Contact us</a>
	      </p>
	    </div>
	  </div>
	</div>
	
      </div><!-- Done row. -->

      <!-- spacer -->
      <div style="height:32px;"></div>
      
      <div class="row">
	
	<div id="our-sources-data" class="col-xl-12">

	  <p>
	  thoughts to organize.
	  
	  also, matrix:
	    https://joinup.ec.europa.eu/software/page/eupl/eupl-compatible-open-source-licences
	    https://www.debian.org/social_contract.html#guidelines
	    also, bring up dfsg vs. gnu
	    https://people.debian.org/~bap/dfsg-faq.html , point 8, 12, etc.
	
	    The freedom to run the program, for any purpose (freedom 0).
	    The freedom to study how the program works, and adapt it to your needs (freedom 1).
	    The freedom to redistribute copies so you can help your neighbor (freedom 2).
	    The freedom to improve the program, and release your improvements to the public, so that the whole community benefits (freedom 3).
	  </p>
	</div>

      </div><!-- Done container. -->
    
    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>
    <script src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js" crossorigin="anonymous"></script>
    <script type="text/javascript">
      var global_data = [{"id":"bgee","source":"BGee","source-link":"http://bgee.org","status":"incomplete","description":"Bgee is a database to retrieve and compare gene expression patterns in multiple animal species, produced from multiple data types (RNA-Seq, Affymetrix, in situ hybridization, and EST data).","data-field":"x-species","data-type":"mouse","data-categories":["expression data"],"data-links":[],"license-type":"copyleft","license-link":"https://github.com/BgeeDB/bgee/blob/master/LICENSE","license-downstream-positive":[],"license-downstream-negative":[],"license-commentary":["It is unclear whether the data or the software is the listed GNU GPL v.3.0; if the former, it may be problematic, if the latter, all rights may be reserved. They do not seem to me looking at the licenses of their own data (http://bgee.org/?page=source)."],"contacts":["Bgee@isb-sib.ch"],"grade":"red","license-issues":[{"criteria":1,"comment":"We are unsure whether the data is covered under the GPLv3 or has all rights reserved."}],"license-commentary-embeddable":"It is unclear whether the data or the software is the listed GNU GPL v.3.0; if the former, it may be problematic, if the latter, all rights may be reserved. They do not seem to me looking at the licenses of their own data (http://bgee.org/?page=source)."},{"id":"biogrid","source":"BioGRID","status":"incomplete","description":"interaction","data-field":"biology","data-type":"x-species","data-categories":["protein-protein interaction"],"data-links":[],"license-type":"source specific","license-link":"http://wiki.thebiogrid.org/doku.php/terms_and_conditions","license-downstream-positive":[],"license-downstream-negative":["All information, data, text, software, music, sound, photographs, graphics, video, messages, or any other materials whatsoever (collectively, “Content”), whether publicly posted or privately transmitted, is the sole responsibility of the person from whom such Content originated. This means that the user, and not BioGRID, is entirely responsible for all Content that he or she uploads, posts, emails or otherwise transmits via the BioGRID Sites.","Use, reproduction, modification, and other intellectual property rights to data stored on the BioGRID Sites will be subject to licensing arrangements that may be approved by BioGRID as applicable to such Content.","You agree not to sell, resell, or offer for any commercial purposes, any portion of the BioGRID Sites, use of the BioGRID Sites, or access to the BioGRID Sites.","Our designated agent to receive notification of claimed infringement is:  Mr. Terry Donaghue Director Technology Transfer and Industrial Liason 600 University Avenue Toronto, ON, Canada donaghue@mshri.on.ca TEL: 416-586-8225 FAX: 416-586-3110"],"license-commentary":[],"contacts":["biogridadmin@gmail.com"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"clinvar","source":"ClinVar","status":"incomplete","description":"ClinVar archives and aggregates information about relationships among variation and human health. ClinVar collects reports of variants found in patient samples, assertions made regarding their clinical significance, information about the submitter, and other supporting data.","data-field":"biology","data-type":"human","data-categories":["disease-gene association","variant-disease association","variant definitions"],"data-links":[],"license-type":"public domain","license-link":"http://www.ncbi.nlm.nih.gov/clinvar/docs/maintenance_use/","license-downstream-positive":["Information that is created by or for the US government on this site is within the public domain. Public domain information on the National Library of Medicine (NLM) Web pages may be freely distributed and copied. However, it is requested that in any subsequent use of this work, NLM be given appropriate acknowledgment."],"license-downstream-negative":["NOTE: This site contains resources which incorporate material contributed or licensed by individuals, companies, or organizations that may be protected by U.S. and foreign copyright laws. These include, but are not limited to PubMed Health (see PubMed Health Copyright and Restrictions Notice), PubMed Central (PMC) (see PMC Copyright Notice), Bookshelf (see Bookshelf Copyright Notice), OMIM (see OMIM Copyright Status), and PubChem. All persons reproducing, redistributing, or making commercial use of this information are expected to adhere to the terms and conditions asserted by the copyright holder. Transmission or reproduction of protected items beyond that allowed by fair use (PDF) as defined in the copyright laws requires the written permission of the copyright owners."],"license-commentary":["TODO: Need to square the reuse on non-PD information."],"contacts":["info@ncbi.nlm.nih.gov"],"grade":"gold","license-commentary-embeddable":"TODO: Need to square the reuse on non-PD information."},{"id":"coriell-institute","source":"Coriell Institute for Medical Research","status":"incomplete","description":"The Coriell Cell Repositories provide essential research reagents to the scientific community by establishing, verifying, maintaining, and distributing cell cultures and DNA derived from cell cultures. These collections, supported by funds from the National Institutes of Health (NIH) and several foundations, are extensively utilized by research scientists around the world. NINDS and NIGMS cell line catalog. NIGMS samples represent a variety of disease states, chromosomal abnormalities, apparently healthy individuals and many distinct human populations. NINDS samples are drawn from subjects with cerebrovascular disease, epilepsy, motor neuron disease, Parkinsonism and Tourette Syndrome, as well as controls.","data-field":"biology","data-type":"human","data-categories":["disease-model association"],"data-links":[],"license-type":"undetermined","license-link":"https://www.coriell.org/legal-notice","license-downstream-positive":[],"license-downstream-negative":["Material on the Coriell Institute website is protected by copyright law. Unless otherwise stated, users may print or download information from this site for personal, non-commercial use only, provided they identify the source of the material (Coriell Institute for Medical Research) and include a statement that the materials are protected by copyright law. Permission to reprint or otherwise reproduce any document in whole or in part is prohibited, unless prior written consent is obtained from the copyright owner (email request to: communications(at)coriell.org)."],"license-commentary":[],"contacts":["communications@coriell.org"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"ctd","source":"Comparative Toxicogenomics Database (CTD)","status":"incomplete","description":"CTD promotes understanding about the effects of environmental chemicals on human health by integrating data from curated scientific literature to describe chemical interactions with genes and proteins, and associations between diseases and chemicals, and diseases and genes/proteins.","data-field":"biology","data-type":"x-species","data-categories":["disease-gene association"],"data-links":[],"license-type":"undetermined","license-link":"http://ctdbase.org/about/legal.jsp","license-downstream-positive":[],"license-downstream-negative":["Any reproduction or use for commercial purpose is prohibited without the prior express written permission of the MDI Biological Laboratory and NC State University."],"license-commentary":[],"contacts":["http://ctdbase.org/help/contact.go"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"encode","source":"ENCODE","status":"incomplete","description":"The Encyclopedia of DNA Elements (ENCODE) Consortium is an international collaboration of research groups funded by the National Human Genome Research Institute (NHGRI). The goal of ENCODE is to build a comprehensive parts list of functional elements in the human genome, including elements that act at the protein and RNA levels, and regulatory elements that control cells and circumstances in which a gene is active.","data-field":"biology","data-type":"genomic resource","data-categories":["genomic elements"],"data-links":[],"license-type":"undetermined","license-link":"https://www.genome.gov/10005107/#al-5","license-downstream-positive":[],"license-downstream-negative":[],"license-commentary":["After 2014 policy: OK.","2008-2014 policy: \"To facilitate this compromise between unrestricted use of the data and unavailability of the data until publication, NHGRI will promote observation of a 9-month period during which resource users may freely use the ENCODE/modENCODE data to design and carry out their own research programs, but not to submit publications or make public presentations that use unpublished ENCODE/modENCODE data without prior consent. The publication and presentation moratorium is expected to extend to all forms of public disclosure, including meeting abstracts, oral presentations, and formal electronic submissions to publicly accessible sites (e.g., public websites, web blogs). (NHGRI recognizes that there may be some exceptions to this blanket request; examples are discussed more fully below.)\""],"contacts":["https://www.genome.gov/10005049/questions-and-feedback/"],"grade":"undetermined","license-commentary-embeddable":"After 2014 policy: OK.<br />2008-2014 policy: \"To facilitate this compromise between unrestricted use of the data and unavailability of the data until publication, NHGRI will promote observation of a 9-month period during which resource users may freely use the ENCODE/modENCODE data to design and carry out their own research programs, but not to submit publications or make public presentations that use unpublished ENCODE/modENCODE data without prior consent. The publication and presentation moratorium is expected to extend to all forms of public disclosure, including meeting abstracts, oral presentations, and formal electronic submissions to publicly accessible sites (e.g., public websites, web blogs). (NHGRI recognizes that there may be some exceptions to this blanket request; examples are discussed more fully below.)\""},{"id":"fantom5","source":"Fantom5","status":"incomplete","description":"We are complex multicellular organisms composed of ~400 distinct cell types. This diversity of cell types allow us to see, think, hear, fight infections etc. yet all of this is encoded in the same genome. The difference between all these cells is what parts of the genome they use – for instance, neurons use different genes than muscle cells, and therefore they work very differently. In FANTOM5, we have systematically investigated exactly what are the sets of genes used in virtually all cell types across the human body, and the genomic regions which determine where the genes are read from. We aim to use this information to build transcriptional regulatory models for every primary cell type that makes up a human.","data-field":"biology","data-type":"human","data-categories":["gene expression"],"data-links":[],"license-type":"CC4","license-link":"http://fantom.gsc.riken.jp/5/","license-downstream-positive":[],"license-downstream-negative":[],"license-commentary":[],"contacts":["fantom-help.gsc@riken.jp"],"grade":"gold","license-commentary-embeddable":""},{"id":"flybase","source":"FlyBase","status":"incomplete","description":"FlyBase is the model organism database providing integrated genetic, genomic, phenomic, and biological data for Drosophila melanogaster.","data-field":"biology","data-type":"MOD","data-categories":["genotype-phenotype association"],"data-links":[],"license-type":"undetermined","license-link":"http://flybase.org/wiki/FlyBase:About#FlyBase_Copyright","license-downstream-positive":[],"license-downstream-negative":["This publication may be copied for non-commercial, scientific uses by individuals or organizations (including for-profit organizations). FlyBase is freely distributed to the scientific community on the understanding that it will not be used for commercial gain by any organization. Any commercial use of this publication, or any parts thereof, is expressly prohibited without permission in writing from the FlyBase consortium."],"license-commentary":[],"contacts":["http://flybase.org/cgi-bin/mailto-fbhelp.html"],"grade":"ugly","license-commentary-embeddable":""},{"id":"gtex","source":"GTEx","status":"incomplete","description":"The Genotype-Tissue Expression (GTEx) project aims to provide to the scientific community a resource with which to study human gene expression and regulation and its relationship to genetic variation. This project will collect and analyze multiple human tissues from donors who are also densely genotyped, to assess genetic variation within their genomes. By analyzing global RNA expression within individual tissues and treating the expression levels of genes as quantitative traits, variations in gene expression that are highly correlated with genetic variation can be identified as expression quantitative trait loci, or eQTLs.","data-field":"biology","data-type":"human","data-categories":["gene expression"],"data-links":[],"license-type":"undetermined","license-link":"http://www.gtexportal.org/home/documentationPage#staticTextPublicationPolicy","license-downstream-positive":["Pilot data set (phs000424.v3.p1) - acceptance of the “Ft. Lauderdale” principles of rapid, pre-publication data release (see Sharing Data from Large-Scale Biological Research Projects: A System of Tripartite Responsibility, 2003). The continued success of rapid pre-publication data release relies on the scientific community to respect the data producer’s interest to publish a full analysis of their data first. Secondary users are asked to refrain from submitting manuscripts describing comprehensive analyses until the Consortium has published their analysis.","May 8, 2015 - No restrictions - main manuscript published.","phs000424.v4.p1 - the dataset is subject to 9 months publication restriction starting from the date of the release; no restrictions after Jan 4, 2015."],"license-downstream-negative":["All datasets from phs00424.v5.p1 forward will follow the NIH GDS policy. This means that once released through dbGaP, there are no restrictions on use or publication. This document and an accompanying table of dataset releases can be found at http://www.gtexportal.org/home/documentationPage.","https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html"],"license-commentary":["Apparently, the data not under the exceptions listed are under the NIH GDS (https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html), which appears to be a well-thought out, but restrictive regime. The NIH GDS section V part B terms are quite restrictive."],"contacts":["http://www.gtexportal.org/home/contact"],"grade":"red","license-commentary-embeddable":"Apparently, the data not under the exceptions listed are under the NIH GDS (https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html), which appears to be a well-thought out, but restrictive regime. The NIH GDS section V part B terms are quite restrictive."},{"id":"hpo","source":"Human Phenotype Ontology (HPO)","status":"incomplete","description":"A curated database of human hereditary syndromes from OMIM, Orphanet, and DECIPHER mapped to classes of the human phenotype ontology. Various meta-attributes such as frequency, references and negations are associated with each annotation. These are presently limited to rare mendelian diseases.","data-field":"biology","data-type":"human","data-categories":["disease-phenotype association"],"data-links":[],"license-type":"undetermined","license-link":"http://human-phenotype-ontology.github.io/license.html","license-downstream-positive":[],"license-downstream-negative":["That the Human Phenotype Ontology Consortium is acknowledged and cited properly.","That neither the content of the HPO file(s) nor the logical relationships embedded within the HPO file(s) be altered in any way."],"license-commentary":["immutability","unclear derivatives"],"contacts":["http://human-phenotype-ontology.github.io/contact.html"],"grade":"yellow","license-commentary-embeddable":"immutability<br />unclear derivatives"},{"id":"impc","source":"International Mouse Phenotyping Consortium (IMPC)","status":"incomplete","description":"The International Mouse Phenotyping Consortium (IMPC) is generating a knockout mouse strain for every protein coding gene by using the embryonic stem cell resource generated by the International Knockout Mouse Consortium (IKMC). Systematic broad-based phenotyping is performed by each IMPC center using standardized procedures found within the International Mouse Phenotyping Resource of Standardised Screens (IMPReSS) resource. Gene-to-phenotype associations are made by a versioned statistical analysis.","data-field":"biology","data-type":"mouse","data-categories":["genotype-phenotype association"],"data-links":[],"license-type":"undetermined","license-link":"http://www.mousephenotype.org/about-impc/additional-information/data-use-agreement","license-downstream-positive":[],"license-downstream-negative":["When the first major Project paper on these analyses is published, then researchers inside and outside the Project are free to present and publish using the Project data for these and other analyses."],"license-commentary":[],"contacts":["http://www.mousephenotype.org/contact-us"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"kegg","source":"Kyoto Encyclopedia of Genes and Genomes (KEGG)","status":"incomplete","description":"KEGG is an integrated database resource consisting of the seventeen main databases including systems, genomic, chemical, and health information.","data-field":"biology","data-type":"genomic resource","data-categories":["gene-pathway association","disease-gene association","orthology"],"data-links":[],"license-type":"undetermined","license-link":"http://www.kegg.jp/kegg/legal.html","license-downstream-positive":[],"license-downstream-negative":["Non-academic users and Academic users intending to use KEGG for commercial purposes are requested to obtain a license agreement through KEGG's exclusive licensing agent, Pathway Solutions, for installation of KEGG at their sites, for distribution or reselling of KEGG data, for software development or any other commercial activities that make use of KEGG, or as end users of any third-party application that requires downloading of KEGG data or access to KEGG data via the KEGG API."],"license-commentary":["Pathway Solutions here: http://www.pathway.jp/"],"contacts":["http://www.genome.jp/feedback/"],"grade":"undetermined","license-commentary-embeddable":"Pathway Solutions here: http://www.pathway.jp/"},{"id":"mgi","source":"Mouse Genome Informatics (MGI)","status":"incomplete","description":"MGI is the international database resource for the laboratory mouse, providing integrated genetic, genomic, and biological data to facilitate the study of human health and disease.","data-field":"biology","data-type":"MOD","data-categories":["genotype-phenotype association","disease-model association","gene expression"],"data-links":[],"license-type":"undetermined","license-link":"http://www.informatics.jax.org/mgihome/other/copyright.shtml","license-downstream-positive":[],"license-downstream-negative":["This software and data are provided to enhance knowledge and encourage progress in the scientific community and are to be used only for research and educational purposes.","Any reproduction or use for commercial purpose is prohibited without the prior express written permission of the Jackson Laboratory."],"license-commentary":[],"contacts":["http://www.informatics.jax.org/mgihome/support/mgi_inbox.shtml"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"mpd","source":"Mouse Phenome Database (MPD)","status":"incomplete","description":"The Mouse Phenome Database is a collaborative standardized collection of measured data on laboratory mouse strains, and includes: baseline phenotype data sets; studies of drug, diet, disease and aging effect; protocols, projects, and publications; and SNP, variation and gene expression studies. MPD collects data for classical inbred strains, other fixed-genotype strains, derived lines and populations that are openly acquirable (strain panel examples). Strains can be from JAX-Mice or from any other vendor that's a recognized breeding source.","data-field":"biology","data-type":"MOD","data-categories":["genotype (strain)-phenotype association"],"data-links":[],"license-type":"undetermined","license-link":"http://phenome.jax.org/db/q?rtn=docs/termsofuse","license-downstream-positive":[],"license-downstream-negative":["MPD data contributors waive copyright and related rights, making their data freely available whereby MPD users may freely build upon, enhance and reuse those data for any purpose without restriction. Scholarly citation norms must be followed for content reuse.","The Jackson Laboratory's web site privacy policy applies to all use of this resource.","The Mouse Phenome Database (MPD) is an NIH-funded data sharing repository and adheres to the NIH Genomic Data Sharing Policy.","Any reproduction of JAX(R) Mice mouse images is prohibited without prior express written permission (see the JAX(R) Mice Terms of Use for details).","Data contributors and data users may not use MPD in any unlawful manner, or in any manner that would impair MPD's services, security or functionality. Automated usage (webcrawlers and similar) must observe each page's \"meta robots\" html tags and space requests by ≥ 2 seconds. We reserve the right to block violators and to take any other action as we deem necessary."],"license-commentary":[],"contacts":["phenome@jax.org"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"mygene-info","source":"MyGene.info","status":"incomplete","description":"MyGene.info provides a simple-to-use REST web services to query/retrieve gene annotation data.","data-field":"biology","data-type":"genomic resource","data-categories":["gene definition"],"data-links":[],"license-type":"undetermined","license-link":"TODO: http://docs.mygene.info/en/v3/terms.html vs. http://mygene.info/terms/","license-downstream-positive":[],"license-downstream-negative":["You also agree that you will ensure that any copies of documents you generate by using MyGene.info shall retain and display all copyright and other proprietary notices contained therein.","Nothing contained herein shall be construed as conferring by implication, estoppel or otherwise any license or right under any patent, trademark or other intellectual property of TSRI or any third party. Except as expressly provided above, nothing contained herein shall be construed as conferring any right or license under any TSRI copyrights."],"license-commentary":[],"contacts":["help@mygene.info"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"ncbi-gene","source":"National Center for Biotechnology Information Gene (NCBI-Gene)","status":"incomplete","description":"NCBI Gene integrates information from a wide range of species, and includes nomenclature, Reference Sequences (RefSeqs), maps, pathways, variations, phenotypes, and links to genome-, phenotype-, and locus-specific resources worldwide. Taxon lists the taxinomic organization of organisms. Pub2Gene serves links between genes and pubmed identifiers where they are mentioned.","data-field":"biology","data-type":"genomic resource","data-categories":["gene definition","taxon definition","gene-publication association"],"data-links":[],"license-type":"undetermined","license-link":"http://www.ncbi.nlm.nih.gov/home/about/policies.shtml","license-downstream-positive":["Information that is created by or for the US government on this site is within the public domain. Public domain information on the National Library of Medicine (NLM) Web pages may be freely distributed and copied. However, it is requested that in any subsequent use of this work, NLM be given appropriate acknowledgment."],"license-downstream-negative":["NOTE: This site contains resources which incorporate material contributed or licensed by individuals, companies, or organizations that may be protected by U.S. and foreign copyright laws. These include, but are not limited to PubMed Health (see PubMed Health Copyright and Restrictions Notice), PubMed Central (PMC) (see PMC Copyright Notice), Bookshelf (see Bookshelf Copyright Notice), OMIM (see OMIM Copyright Status), and PubChem. All persons reproducing, redistributing, or making commercial use of this information are expected to adhere to the terms and conditions asserted by the copyright holder. Transmission or reproduction of protected items beyond that allowed by fair use (PDF) as defined in the copyright laws requires the written permission of the copyright owners."],"license-commentary":["TODO: Need to square the reuse on non-PD information."],"contacts":["info@ncbi.nlm.nih.gov"],"grade":"gold","license-commentary-embeddable":"TODO: Need to square the reuse on non-PD information."},{"id":"omia","source":"Online Mendelian Inheritance in Animals (OMIA)","status":"incomplete","description":"Online Mendelian Inheritance in Animals (OMIA) is a catalogue/compendium of inherited disorders, other (single-locus) traits, and genes in 215 (non-model) animal species.","data-field":"biology","data-type":"veterinary x-species","data-categories":["gene-disease association"],"data-links":[],"license-type":"undetermined","license-link":"http://sydney.edu.au/disclaimer.html","license-downstream-positive":[],"license-downstream-negative":["Individuals may copy a reasonable portion of the material on this site under the fair dealing provisions of the Copyright Act 1968 provided it is used for research or study or for criticism or review. Educational institutions may reproduce limited amounts of material from this site under Part VB of the Copyright Act","Apart from these specified uses material from this site may not be reproduced for commercial purposes without prior written permission of the University of Sydney."],"license-commentary":[],"contacts":[""],"grade":"undetermined","license-commentary-embeddable":""},{"id":"omim","source":"Mendelian Inheritance in Man relationship to HPO construction and maintenance (OMIM)","status":"incomplete","description":"OMIM is a comprehensive, authoritative compendium of human genes and genetic phenotypes with full-text, referenced overviews that contains information on all known mendelian disorders and over 12,000 genes. OMIM focuses on the relationship between phenotype and genotype.","data-field":"biology","data-type":"human","data-categories":["disease-phenotype association","gene-disease association","variant-disease association"],"data-links":[],"license-type":"undetermined","license-link":"http://www.omim.org/help/agreement","license-downstream-positive":[],"license-downstream-negative":["Use of OMIM.org is provided free of charge to any individual for personal use, for educational or scholarly use, or for research purposes through the front end of the database."],"license-commentary":[],"contacts":["ahamosh@jhmi.edu"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"orphanet","source":"ORPHANET","status":"incomplete","description":"Orphanet provides reference information on rare diseases and orphan drugs to help improve the diagnosis, care and treatment of patients with rare diseases.","data-field":"biology","data-type":"human","data-categories":["disease-gene association","disease-phenotype association","ontology"],"data-links":[],"license-type":"undetermined","license-link":"http://www.orpha.net/consor/cgi-bin/Education_AboutOrphanet.php?lng=EN&stapage=CGU","license-downstream-positive":[],"license-downstream-negative":["No element or part of the site (including, but not limited to, the database and the information contained within) can be copied, translated, reproduced, exploited, published, electronically stocked or redistributed for commercial use without prior written consent.","Non-profit organisations are authorised to copy and distribute information from the site, as long as this information is neither modified nor sold. In addition to this, the terms of the intellectual property rights and the source of the information must be provided."],"license-commentary":[],"contacts":["contact.orphanet@inserm.fr"],"grade":"undetermined","license-commentary-embeddable":""},{"id":"panther","source":"Protein ANalysis THrough Evolutionary Relationships Classification System (PANTHER)","status":"incomplete","description":"The PANTHER (Protein ANalysis THrough Evolutionary Relationships) Classification System was designed to classify proteins (and their genes) according to evolutionary family/subfamily, molecular function, biological process, and pathway. The PANTHER Classifications are the result of human curation as well as sophisticated bioinformatics algorithms.","data-field":"biology","data-type":"genomic resource","data-categories":["orthology"],"data-links":[],"license-type":"ARR","license-link":"http://pantherdb.org/terms/disclaimer.jsp ?","license-downstream-positive":[],"license-downstream-negative":[],"license-commentary":["TODO: Reach out to Paul--according to the site he is the sole copyright holder. Considering the disclaimer, I believe the intention is something BSD-ish, but the literal read is all right reserved."],"contacts":["feedback@pantherdb.org","pdthomas@usc.edu"],"grade":"undetermined","license-commentary-embeddable":"TODO: Reach out to Paul--according to the site he is the sole copyright holder. Considering the disclaimer, I believe the intention is something BSD-ish, but the literal read is all right reserved."},{"id":"reactome","source":"Reactome","status":"incomplete","description":"Reactome is a free, open-source, curated and peer reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic research, genome analysis, modeling, systems biology and education.","data-field":"biology","data-type":"pathway x-species","data-categories":["pathway data"],"data-links":["http://www.reactome.org/pages/download-data/"],"license-type":"CC4","license-link":"http://www.reactome.org/pages/about/license-agreement/","license-downstream-positive":[],"license-downstream-negative":[],"license-commentary":["There is a comment in there about KEGG data in Reactome. Thus Reactome FI Network data is not available."],"contacts":["help@reactome.org"],"grade":"gold","license-commentary-embeddable":"There is a comment in there about KEGG data in Reactome. Thus Reactome FI Network data is not available."},{"id":"wormbase","source":"WormBase database of nematode biology (WormBase)","status":"incomplete","description":"WormBase is an international consortium dedicated to providing the research community with accurate, current, accessible information concerning the genetics, genomics and biology of C. elegans and related nematodes.","data-field":"biology","data-type":"MOD","data-categories":["allele-phenotype association"],"data-links":[],"license-type":"undetermined","license-link":"http://www.wormbase.org/about/policies#0--10","license-downstream-positive":["In consideration of other users, we request that any script -- whether it be mining the web interface or directly interacting with the underlying databases -- send no more than one request every three seconds."],"license-downstream-negative":[],"license-commentary":["This has to do with hitting their servers to much and being too big a load; this may not apply."],"contacts":["help@wormbase.org"],"grade":"undetermined","license-commentary-embeddable":"This has to do with hitting their servers to much and being too big a load; this may not apply."},{"id":"zfin","source":"Zebrafish Information Network (ZFIN)","status":"incomplete","description":"The Zebrafish Information Resource is the community database resource for the laboratory use of zebrafish which develops and supports integrated zebrafish genetic, genomic and developmental information, maintains the definitive reference data sets of zebrafish research information toward facilitation of the use of zebrafish as a model for human biology.","data-field":"biology","data-type":"MOD","data-categories":["genotype-phenotype association","experimental reagents (morpholinos)"],"data-links":[],"license-type":"undetermined","license-link":"http://zfin.org/warranty.html","license-downstream-positive":[],"license-downstream-negative":["ZFIN Software and Data are provided to enhance knowledge and encourage progress in the scientific community, and are to be used only for research and educational purposes. ZFIN reserves all rights not expressly granted. Further, ZFIN retains all rights, title, and interest in and to the Software and Data. Any reproduction or use for a commercial purpose is prohibited without the prior express written permission of ZFIN. Distribution of the Software or Data to any third party, without a separate written agreement with ZFIN, is prohibited."],"license-commentary":[],"contacts":["jknight@zfin.org"],"grade":"undetermined","license-commentary-embeddable":""}];
      jQuery(document).ready(function(){
         console.log('Ready...');
         jQuery("#sourcestable").DataTable(
                    {autoWidth: true}
         );
         console.log('...done.');
         //{autoWidth: true, "order": [[3, "desc"], [0, "asc"]]}
         // For license commentary popups.
         jQuery(function () {
            jQuery('[data-toggle="popover"]').popover()
         });
      });
    </script>

  </body>
  
</html>
